config/insert_config_splicemachine_4.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<parameters>
+	
+	<!-- Connection details -->
+	<dbtype>splicemachine</dbtype>
+	<driver>com.splicemachine.db.jdbc.ClientDriver</driver>
+	<DBUrl>jdbc:splice://localhost:1527/splicedb</DBUrl>
+	<username>htap</username>
+	<password>htapuser</password>
+	<isolation>TRANSACTION_SERIALIZABLE</isolation>
+	
+	<!-- Scale factor is the number of warehouses in TPCC -->
+	<scalefactor>25</scalefactor>
+
+	<!-- The workload -->
+	<terminals>4</terminals>
+	<works>
+		<work>
+			<!-- set serial to true for testing each transaction once - be sure to set 0 for time value --> 
+			<!--<serial>true</serial>-->
+
+			<!--<warmup>300</warmup>-->
+			<time>3600</time>
+ 			<active_terminals bench="chbenchmark">4</active_terminals>
+			<rate bench="chbenchmark">unlimited</rate>
+			<weights bench="chbenchmark">100, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5</weights>
+		</work>
+	</works>
+	
+	<!-- TPCC specific -->  
+	<transactiontypes bench="chbenchmark">
+		<transactiontype>
+			<name>INSERT</name>
+		</transactiontype>
+		<transactiontype>
+			<name>Q1</name>
+		</transactiontype>
+		<transactiontype>
+			<name>Q2</name>
+		</transactiontype>
+		<transactiontype>
+			<name>Q3</name>
+		</transactiontype>
+		<transactiontype>
+			<name>Q4</name>
+		</transactiontype>
+		<transactiontype>
+			<name>Q5</name>
+		</transactiontype>
+		<transactiontype>
+			<name>Q6</name>
+		</transactiontype>
+		<transactiontype>
+			<name>Q7</name>
+		</transactiontype>
+		<!--<transactiontype>
```

Comment:
@OlegMazurov under what condition will Q8 be re-enabled? Do you have the tracking JIRA for the Q8 problem?

Reply:
I don't know. @rodbutters ?

Reply:
I filed DB-7652 on Q8 failure. It remains disabled in all configs for now.